### PR TITLE
Add app info to iframe

### PR DESCRIPTION
**Pull request by xitronix**

### CODE REVIEW

File: universal-login-web3/src/ui/utils/getValueFromSearch.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import {MissingParameter} from './errors';
+
+export function getValueFromSearch(search: string, paramName: string) {
```
Comment (Marik-D) [inline universal-login-web3/src/ui/utils/getValueFromSearch.ts]: What about having a function `parseSearch(serach: string): Record<string, string>` that will return an object with all keys and values parsed from search

File: universal-login-provider/src/ULIFrameProvider.ts [hunk 2]
```diff
@@ -19,7 +21,9 @@ export class ULIFrameProvider {
     config = DEFAULT_CONFIG,
     enablePicker = false,
   ) {
-    this.iframe = createIFrame(config.backendUrl + (enablePicker ? '?picker' : ''));
+    const applicationInfo = getApplicationInfoFromDocument();
+    const uriComponent = createUriComponent(config.backendUrl, applicationInfo, enablePicker);
```
Comment (Marik-D) [inline universal-login-provider/src/ULIFrameProvider.ts]: Rename this function to `buildIframeUrl`

Comment (Marik-D) [reply universal-login-provider/src/ULIFrameProvider.ts]: And variable to `iframeUrl`

File: universal-login-web3/src/ui/iframe.tsx [hunk 3]
```diff
@@ -52,8 +52,11 @@ function initPicker() {
   sendReadySignal(bridge);
 }
 
-if (window.location.search.includes('picker')) {
-  initPicker();
+const parsedQuery = parseQuery(window.location.search);
+const applicationInfo = JSON.parse(parsedQuery.applicationInfo) as ApplicationInfo;
```
Comment (Marik-D) [inline universal-login-web3/src/ui/iframe.tsx]: Please use sanitizer here

Comment (xitronix) [reply universal-login-web3/src/ui/iframe.tsx]: 👍

File: universal-login-commons/src/core/utils/sanitizers/asApplicationInfo.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import {Sanitizer, asObject, asString} from '@restless/sanitizers';
+import {ApplicationInfo} from '../../models/notifications';
+
+export const asApplicationInfo: Sanitizer<ApplicationInfo> = asObject<ApplicationInfo>({
```
Comment (Marik-D) [inline universal-login-commons/src/core/utils/sanitizers/asApplicationInfo.ts]:
```suggestion
export const asApplicationInfo = asObject<ApplicationInfo>({
```

File: universal-login-commons/src/core/utils/sanitizers/asApplicationInfo.ts [hunk 5]
```diff
@@ -0,0 +1,8 @@
+import {Sanitizer, asObject, asString} from '@restless/sanitizers';
```
Comment (Marik-D) [inline universal-login-commons/src/core/utils/sanitizers/asApplicationInfo.ts]:
```suggestion
import {asObject, asString} from '@restless/sanitizers';
```

**Pull request merged**